projects/igniteui-angular/src/lib/icon/icon.broadcast.service.ts

```diff
@@ -0,0 +1,79 @@
+import { Injectable, Optional } from '@angular/core';
```

Comment:
Should this be in the Elements project source?
Or did we consider the option for this to be available for Angular apps, though with the current functionality I'm not sure that'd be of much use since it's not two-way yet and the registries parity is also not 1:1 yet.

Reply:
I was thinking of making it optional, so that if someone wants to inject it for some reason into an angular project they could. But we could just have it in elements if that makes more sense.